src/features/instanceData/getInstanceDataSagas.ts

```diff
@@ -16,8 +16,10 @@ export function* getInstanceDataSaga({ payload: { instanceId } }: PayloadAction<
     if (error.response && error.response.status === 403 && error.response.data) {
       const reqAuthLevel = error.response.data.RequiredAuthenticationLevel;
       if (reqAuthLevel) {
-        putWithoutConfig(invalidateCookieUrl);
+        putWithoutConfig(invalidateCookieUrl).then();
```

Comment:
Does the `.then()` do anything here?

Reply:
It just makes WebStorm ignore the unhandled/un-awaited Promise returned from the PUT... In v4 I've taken the chance to rewrite this thing properly.